ClientApp/viking-vault/src/UserDashboard/AccountPage.tsx

```diff
@@ -18,12 +21,17 @@ interface IAccountBalance{
 interface IAccountState{
```

Comment:
It seems that some of those properties in this state can be separated further into their own objects.
like redirect,  openBlockCardResponseModal, blockCardResponseMessage. firstName dont't really belong together